Add UPIC-Emma to the list of codes

RemiLehe opened the pull request:
UPIC Emma recently added an openPMD writer (see [this PR](https://github.com/UCLA-Plasma-Simulation-Group/upic-emma-2.0/pull/4/files)).

Therefore, the present PR adds UPIC to the list of code.
@tsung1029 : Could you confirm that the information is correct? For the `domain`/description, I used the header of the file `upic-emma.f90`.

RemiLehe committed:
Added UPIC-emma

ax3l commented on README.md:
```diff
@@ -60,6 +60,13 @@ the work provided in those for our format handling.
 
+- UPIC-Emma (UCLA, USA)
+  - domain: 2-1/2D electromagnetic particle-in-cell code
+  - [repository](https://github.com/UCLA-Plasma-Simulation-Group/upic-emma-2.0)
```
@RemiLehe can you please add the license in brackets as well?
Do we know the (abbreviated) name of the license in [UPIC-Emma](https://github.com/UCLA-Plasma-Simulation-Group/upic-emma-2.0/blob/master/license)?

 Just for curiosity, is this an [OSI-approved](https://opensource.org/licenses) license? Looks like a mix of a new-BSD/MIT-style license and a [CC-BY-NC](https://creativecommons.org/licenses/by-nc/4.0/) restriction.

ax3l commented on README.md:
```diff
@@ -125,7 +132,7 @@ Please check the individual repositories and feel free to contribute.
 - [Fortran openPMD writer](https://github.com/UCLA-Plasma-Simulation-Group/Fortran-OpenPMD-File-Writers) (UCLA, USA)
-  - domain: Fortran 2003 HDF5 File Writers in openPMD Standard 
+  - domain: Fortran 2003 HDF5 File Writers in openPMD Standard
   - [repository](https://github.com/UCLA-Plasma-Simulation-Group/Fortran-OpenPMD-File-Writers)
```
we also forgot to add the license here. looks like it's the same as UPIC-Emma's

ax3l reviewed (changes requested):
Thanks for the addition!
Please add the licenses as well :)

tsung1029 commented:
Hi Everyone:

UPIC-EMMA supports OpenPMD.  I will look into the type of license that it has.  It is the same one that we all copy/paste for PICKSC.  —   Frank


> On May 17, 2018, at 2:10 AM, Axel Huebl <notifications@github.com> wrote:
> 
> @ax3l requested changes on this pull request.
> 
> Thanks for the addition!
> Please add the licenses as well :)
> 
> In README.md <https://github.com/openPMD/openPMD-projects/pull/15#discussion_r188887606>:
> 
> > @@ -60,6 +60,13 @@ the work provided in those for our format handling.
>    - original author: A Grund @Flamefire
>    - status: implemented (base standard for X-ray laser & density profiles)
>  
> +- UPIC-Emma (UCLA, USA)
> +  - domain: 2-1/2D electromagnetic particle-in-cell code
> +  - [repository](https://github.com/UCLA-Plasma-Simulation-Group/upic-emma-2.0)
> @RemiLehe <https://github.com/RemiLehe> can you please add the license in brackets as well?
> Do we know the (abbreviated) name of the license in UPIC-Emma <https://github.com/UCLA-Plasma-Simulation-Group/upic-emma-2.0/blob/master/license>?
> 
> Just for curiosity, is this an OSI-approved <https://opensource.org/licenses> license? Looks like a mix of a new-BSD/MIT-style license and a CC-BY-NC <https://creativecommons.org/licenses/by-nc/4.0/> restriction.
> 
> In README.md <https://github.com/openPMD/openPMD-projects/pull/15#discussion_r188888389>:
> 
> > @@ -125,7 +132,7 @@ Please check the individual repositories and feel free to contribute.
>          - 1.4.0+: full API available to fulfill the standard (read+write)
>  
>  - [Fortran openPMD writer](https://github.com/UCLA-Plasma-Simulation-Group/Fortran-OpenPMD-File-Writers) (UCLA, USA)
> -  - domain: Fortran 2003 HDF5 File Writers in openPMD Standard 
> +  - domain: Fortran 2003 HDF5 File Writers in openPMD Standard
>    - [repository](https://github.com/UCLA-Plasma-Simulation-Group/Fortran-OpenPMD-File-Writers)
> we also forgot to add the license here. looks like it's the same as UPIC-Emma's
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub <https://github.com/openPMD/openPMD-projects/pull/15#pullrequestreview-120953920>, or mute the thread <https://github.com/notifications/unsubscribe-auth/AGMB9IMoK0SUHYT78AMSWCi-wJt1yRicks5tzT6fgaJpZM4UBkQw>.
>

ax3l commented:
Somehow this last mail has not reached GitHub:

> FYI, Axel is correct.   This is the standard license for all of the PICKSC software and I was not aware of its origin before.  Thanks for clearing that up.
>
> Frank

I will update the PR and abbreviate the license as `UCLA-NC`.

ax3l committed:
Add UCLA Software license

License currently used in all PICKSC software.
See for example
  https://github.com/UCLA-Plasma-Simulation-Group/upic-emma-2.0/blob/045dfbab3ffc1b4f5a3b63e06e87d61129e1d1ee/license
.
